### ISSUE TITLE
possible regexp-regression in commit 413f3ef5f4ac11a3d1bb9caf77964a8e6b3276f3

### ISSUE THREAD

**Issue by kaizhu256** [douglascrockford/JSLint#271]
the following code was valid before commit 413f3ef5f4ac11a3d1bb9caf77964a8e6b3276f3

```js
let regexp = /a{2,}\u0000/;
```

now it gives incorrect warning about unclosed curly-brace:

```js
[JSLint was unable to finish] - Expected '}' and instead saw '/'.
```

![image](https://user-images.githubusercontent.com/280571/92528649-bb978000-f1ee-11ea-8276-c0774d552469.png)

traced the incorrect-warning to https://github.com/douglascrockford/JSLint/blob/413f3ef5f4ac11a3d1bb9caf77964a8e6b3276f3/jslint.js#L972 but not sure how to fix it.

**Comment by douglascrockford** [comment]
Thanks.
